crates/bevy_gltf/src/loader.rs

```diff
@@ -177,6 +178,20 @@ async fn load_gltf<'a, 'b, 'c>(
     settings: &'b GltfLoaderSettings,
 ) -> Result<Gltf, GltfError> {
     let gltf = gltf::Gltf::from_slice(bytes)?;
+    let file_name = load_context
```

Comment:
I tried this PR now locally, and I get logs such as:

```
2024-02-16T08:43:52.892063Z DEBUG bevy_gltf::loader: Missing vertex tangents for scene.gltf, computing them using the mikktspace algorithm. Consider using a tool such as Blender to pre-compute the tangents.
2024-02-16T08:43:52.920969Z DEBUG bevy_gltf::loader: Missing vertex tangents for scene.gltf, computing them using the mikktspace algorithm. Consider using a tool such as Blender to pre-compute the tangents.
2024-02-16T08:43:52.951979Z DEBUG bevy_gltf::loader: Missing vertex tangents for scene.gltf, computing them using the mikktspace algorithm. Consider using a tool such as Blender to pre-compute the tangents.
2024-02-16T08:43:52.981170Z DEBUG bevy_gltf::loader: Missing vertex tangents for scene.gltf, computing them using the mikktspace algorithm. Consider using a tool such as Blender to pre-compute the tangents.
2024-02-16T08:43:53.009484Z DEBUG bevy_gltf::loader: Missing vertex tangents for scene.gltf, computing them using the mikktspace algorithm. Consider using a tool such as Blender to pre-compute the tangents.
```

I think it's likely that other users also have a lot of non-unique GLTF names.
E.g. some asset stores often have such generic names when you buy models from them.

So perhaps using the load context path in full is better than just the file name.